snmp/datadog_checks/snmp/data/default_profiles/alcatel-lucent-ent.yaml

```diff
@@ -239,15 +258,24 @@ metrics:
           10: port
           11: stack
           12: cpu
-      - column:
+          13: energy_object
+          14: battery
+          15: storage_drive
+      - MIB: ENTITY-MIB
+        table: entPhysicalTable
```

Comment:
If I'm understanding correctly you added MIB/table since we are referring to a external tables.

I think it might be better to use comment instead of actual fields for MIB/table since those fields are not documented.